Data/DataModel/Feedback.cs

```diff
@@ -43,6 +43,7 @@ public Feedback()
         /// <summary>
         /// The feedback text submitted by the user.
         /// </summary>
+        [MaxLength(500)]
```

Comment:
This might be short for feedback and it's only in betas, so it might be nice to leave this unlimited or at least put in a very long constraint

Reply:
This is now 5000